.gitignore

```diff
@@ -10,6 +10,7 @@ tests/e2e/screenshots
 .DS_Store
 /tests/e2e/videos/
 /tests/e2e/screenshots/
+.idea
```

Comment:
consider set up a global .gitignore to don't be distracted by this in future https://gist.github.com/subfuzion/db7f57fff2fb6998a16c